metadata-ingestion/recipe_overview.md

```diff
@@ -0,0 +1,84 @@
+# Recipes
+
+A recipe is the main configuration file for metadata ingestion. It tells our ingestion scripts where to pull data from (source) and where to put it (sink).
+
+## Configuring Recipe
+
+The basic form of the recipe file consists of:
+
+- `source`, which contains the configuration of the data source. (See [Sources](source_overview.md))
+- `sink`, which defines the destination of the metadata (See [Sinks](sink_overview.md))
+
+Here's a simple recipe that pulls metadata from MSSQL (source) and puts it into the default sink (datahub rest).
+
+```yaml
+# The simplest recipe that pulls metadata from MSSQL and puts it into DataHub
+# using the Rest API.
+source:
+  type: mssql
+  config:
+    username: sa
+    password: ${MSSQL_PASSWORD}
+    database: DemoData
+# sink section omitted as we want to use the default datahub-rest sink
+sink:
+  type: "datahub-rest"
+  config:
+    server: "http://localhost:8080"
+```
+
+A number of recipes are included in the [examples/recipes](./examples/recipes) directory. For full info and context on each source and sink, see the pages described in the [table of plugins](../docs/cli.md#installing-plugins).
+
+:::note One Source/Sink for One Recipe!
+Note that one recipe file can only have 1 source and 1 sink. If you want multiple sources then you will need multiple recipe files.
+:::
+
+## Running Recipe
+
+Running this recipe is as simple as:
+
+```shell
+datahub ingest -c recipe.dhub.yaml
+```
+
+For a detailed guide on running recipes via CLI, please refer to [CLI Ingestion Guide](cli-ingestion.md).
+
+### Handling Sensitive Information in Recipes
```

Comment:
let's put this all in an `## Advanced` section or something?

mainly that this heading nesting doesn't make sense
<img width="299" alt="image" src="https://github.com/datahub-project/datahub/assets/12566801/1edb297d-e1e3-4dad-a8d0-e5c3992dfc92">